frontend/src/__tests__/cypress/cypress/pages/modelRegistry.ts

```diff
@@ -6,6 +6,11 @@ class ModelRegistry {
     this.waitLanding();
   }
 
+  visitModelRegistry() {
```

Comment:
You can use `visit` function below and change it's path from `/modelRegistry${modelRegistry}` to `/modelRegistry`